nixos/modules/programs/minipro.nix

```
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.minipro;
+in
+{
+  options = {
+    programs.minipro = {
+      enable = mkEnableOption (mdDoc "minipro") // {
+        description = mdDoc ''
+          Installs minipro and its udev rules.
+          Users of the `plugdev` group can interact with connected MiniPRO chip programmers.
+        '';
+      };
+
+      package = mkOption {
+        default = pkgs.minipro;
+        defaultText = literalExpression "pkgs.minipro";
+        description = mdDoc "Minipro package to use.";
+        type = types.package;
+      };
```

So I learned about `mkPackageOptionMD` a few days ago. Maybe this could use that?

```suggestion
      package = mkPackageOptionMD pkgs "minipro" { };
```

I was coming back to this PR to merge, but it looks like there's a merge conflict in the release notes now too (meant to come back to this earlier, oops)

---

Did not know about `mkPackageOptionMD`, definitely noted!

Also went ahead and fixed the merge conflict.